src/frontends/pytorch/src/op/int.cpp

```diff
@@ -12,10 +12,10 @@ namespace pytorch {
 namespace op {
 
 OutputVector translate_int(NodeContext& context) {
-    return {context.mark_node(std::make_shared<opset10::Convert>(context.get_input(0), element::i64))};
+    return {context.mark_node(std::make_shared<opset10::Convert>(context.get_input(0), element::i32))};
```

Comment:
Alternative to it for view would be to `Convert` all inputs to `prim::ListConstruct` into common precision.